server/connectors/bitbucket_connector/webhook_secret.py

```
@@ -0,0 +1,118 @@
+                      AND bitbucket_webhook_secret_ref IS NULL""",
+                (ref, org_id),
+            )
+            raced = cur.rowcount == 0
+        conn.commit()
+    if raced:
+        # A concurrent enable won the ref write — use theirs, drop ours.
+        winner = get_webhook_secret(org_id)
+        if winner:
+            try:
+                SecretRefManager().delete_secret(ref)
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)